charts/generic-service/values.yaml

```diff
@@ -144,6 +144,8 @@ tracing:
   enabled: false
   probability: 1
   class: ''
+  # By default if tracing is enabled, a sidecar is injected without TLS. 
```

Comment:
Please add this comment to README.md and values.schema.json instead.

Reply:
Done.